docs/6.-Engineering/Quality-Standards/Coding-Conventions.md

```diff
@@ -21,6 +21,10 @@ For constructing a larger string e.g. within a loop, use `StringBuilder` instead
 
 Unit and Integration tests should make use [Shouldly](https://github.com/shouldly/shouldly#readme) instead of `Assert.That` to make assertions more readable.
 
+
+# Conventions for packages/dependencies
+When including packages/dependencies (internal or external) allways use a exact package version, do not specify app version with wildcards or a minimum version of an package.
```

Comment:
```suggestion
When including packages/dependencies (internal or external) always use a exact package version, do not specify app version with wildcards or a minimum version of an package.
```

Reply:
I think 'd put this in the high level quality standards rather than make it dotnet specific. 

Possibly you could detail in the dotnet and JavaScript specific coding conventions *how* we state exact packages.

Also use the word must e.g. When including packages/dependencies (internal or external) **developers must** always

Reply:
Hi, integrated these changes in the latest commit, could you take a look again? 🤔

Reply:
right, should have used the document/package-inclusion-convetions branch name
My bad :)